### PR TITLE
LPS118531 - regen snapshots

### PR DESCRIPTION
**Pull request by brunofariasdeo**
https://issues.liferay.com/projects/LPS118531

### CODE REVIEW

**Comment by brunofariasdeo** [comment]
ci:test:sf

**Comment by brunofariasdeo** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/test/js/components/card-shortcut/CardShortcut.es.js [hunk 1]
```diff
@@ -70,10 +53,8 @@ describe('CardShortcut', () => {
 	it('anchors are created for each field', () => {
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/test/js/components/card-shortcut/CardShortcut.es.js:53]: @brunofariasdeo, please update the test's description according to your changes. Thanks.

**Review by natocesarrego — CHANGES_REQUESTED** [review]

**Comment by brunofariasdeo** [comment]
ci:test:sf

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/test/js/components/card-shortcut/CardShortcut.es.js (hunk 1 above)
Comment (brunofariasdeo) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/test/js/components/card-shortcut/CardShortcut.es.js]: @natocesarrego, I have updated it accordingly to my changes, I have missed the fact that 'anchor' stands for 'a' tag.

**Comment by brunofariasdeo** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/test/js/components/card-shortcut/CardShortcut.es.js (hunk 1 above)
Comment (natocesarrego) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/test/js/components/card-shortcut/CardShortcut.es.js]: No problem. Thanks. 🙂

**Comment by natocesarrego** [comment]
ci:forward

**Pull request closed without merging**